platform/ui/src/components/ContextMenuMeasurements/ContextMenuMeasurements.jsx

```diff
@@ -0,0 +1,44 @@
+import { ContextMenu } from '@ohif/ui';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const ContextMenuMeasurements = ({
+  onGetMenuItems,
+  onSetLabel,
+  onClose,
+  onDelete,
+}) => {
+  const defaultMenuItems = [
+    {
+      label: 'Delete measurement',
+      actionType: 'Delete',
+      action: item => {
+        onDelete(item);
+        onClose();
+      },
+      value: {}
+    },
+    {
+      label: 'Relabel',
```

Comment:
In previous versions, it said "Relabel" if the measurement already had a label, and "Add Label" otherwise. If we don't have that behaviour, I think it should Add Label by default.